app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
```

Comment:
- [ ] Based on the code, your user model looks fine, but unfortunately I wasn't able to test your app locally because I cannot pull the changes due to this error. Please add additional instructions in your readme file if anything extra needs to be added.
![Screenshot from 2021-11-15 22-54-34](https://user-images.githubusercontent.com/62705947/141859991-e28f4d22-643a-4223-afb5-6349792ae4b5.png)

Reply:
Hello Gzim! Please try to use git pull origin devise --allow-unrelated-histories